src/main/kotlin/com/personio/synthetics/client/BrowserTest.kt

```diff
@@ -37,9 +41,12 @@ fun getCredentialsProvider(): CredentialsProvider {
 }
 
 /**
- * Synthetic browser test client for the API calls with the Datadog
+ * Synthetic browser test client for the API calls with the Datadog.
+ *
+ * Deprecation notice: In future releases this class will be replaced by
+ * SyntheticTestBuilder (or similar).
  */
-class BrowserTest(testName: String, private val syntheticsApiClient: SyntheticsApiClient, private val defaultSettings: Defaults = Config.testConfig.defaults) : SyntheticsBrowserTest() {
+@Deprecated(message="Planned for removal in 3.x.x") class BrowserTest(testName: String, private val syntheticsApiClient: SyntheticsApiClient, private val defaultSettings: Defaults = Config.testConfig.defaults) : SyntheticsBrowserTest() {
```

Comment:
Perhaps, it's better to move class from a new line?

Reply:
Resolved in another commit